server/services/deployment.ts

```diff
@@ -72,15 +77,6 @@ export async function findDeploymentsByModel(user: UserDoc, model: ModelDoc, opt
   return filterDeployment(user, await deployments)
 }
 
-export async function findDeploymentsByVersion(user: UserDoc, version: VersionDoc, opts?: GetDeploymentOptions) {
```

Comment:
Was this removed for being dead/unused code?

Reply:
Yes, I couldn't find any references to it and I don't think we include version info in deployment records anymore